lib/p5.play.js

```diff
@@ -171,8 +171,8 @@ if (!p5.prototype.originalCreateCanvas_) {
 if (!p5.prototype.originalEllipse_) {
   p5.prototype.originalEllipse_ = p5.prototype.ellipse;
   p5.prototype.ellipse = function(x, y, w, h) {
-    w = (w) ? w : 50;
-    h = (w && !h) ? w : h;
+    w = (w === undefined) ? 50 : w;
```

Comment:
if we use `==` instead of `===` here, we'll catch the case where `w` is null as well. this is arguable as a useful feature of javascript, but good for us in this case?